modules/Kconfig.cmsis_nn

```diff
@@ -0,0 +1,61 @@
+# Copyright (c) 2021, Commonwealth Scientific and Industrial Research
+# Organisation (CSIRO) ABN 41 687 119 230.
+# SPDX-License-Identifier: Apache-2.0
+
+comment "CMSIS-NN Components"
+
+config CMSIS_NN_ACTIVATION
```

Comment:
Do these components have any inter-component dependencies? If so, it would be nice to add `imply` to the relevant configs.

For example:
https://github.com/zephyrproject-rtos/zephyr/blob/4bc73dd072e9257fba7aed1c05310fe555ae2bf6/modules/Kconfig.cmsis_dsp#L29-L31

Reply:
Good point, a few of them do rely on the support functions.
I have added these dependencies.